src/server/generic_family_test.cc

```diff
@@ -478,7 +478,13 @@ TEST_F(GenericFamilyTest, Sort) {
 
   Run({"set", "foo", "bar"});
   ASSERT_THAT(Run({"sort", "foo"}), ErrArg("WRONGTYPE "));
-  ;
+
+  Run({"rpush", "list-3", ""});
+  ASSERT_THAT(Run({"sort", "list-3"}), "");
+
+  Run({"rpush", "list-3", "2", "0", "", "-0.14", "0.12", "-0", "nan", "-123123", "7654"});
+  ASSERT_THAT(Run({"sort", "list-3"}).GetVec(),
+              ElementsAre("-123123", "-0.14", "", "", "-0", "0", "0.12", "2", "7654", "nan"));
```

Comment:
Also one more question here. `nan` is a string in `context` of `set` command and not `nan` like `not a number for floats`. So the question is, does redis/valkey do the same here ? Because for `lexicographical sorting` we need `SORT list ALPHA`

Reply:
yes I've taken this behavior from Redis

Reply:
@BorysTheDev Yep I verified with `valkey` it does not work

Reply:
@BorysTheDev run the following:

```
rpush mylist 2 0 "" -0.14 0.12 -0 "nan" -123123 7654
sort mylist
```
and you will get

```
ERR One or more scores can't be converted into double
```

Reply:
I've got these cases from fakeredis tests and checked Redis. Why don't you want to merge?

Reply:
Because this query is not working on redis or valkey. You can try it yourself

Reply:
it works for redis

Reply:
Ok I will remove Nan for now